README.md

```diff
@@ -29,12 +29,18 @@ To run the tutorials or the test suite locally, run the following to install the
 pip install -e .[dev]
 ```
 
+To make sure you have a good code quality, please install pre-commit, it will trigger useful linting checks:
```

Comment:
nit: reword: `We use pre-commit hooks to enforce some code linting, you can install pre-commit with Python pip.`